drafts/build_a_dao_on_fuel/draft.md

```diff
@@ -0,0 +1,1459 @@
+[note]: <> ("note to self and file editor, there are specific comments left throughout this file, for the sake of understanding the author's reason for using a particular method of approach; pls feel free to delete once acknowledged or reply edit with a better approach 🥂")
+
+# How to Build a DAO on the Fuel Blockchain.
+
+## Introduction
+
+A Decentralized Autonomous Organization (DAO) is considered an essential blockchain application.
+
+It is a Decentralized application that users or organizations use to automate governance execution of their projects and communities
+It is an automated program represented by rules encoded in a computer that is transparently controlled by the organization members and not influenced by any central authority.
```

Comment:
It's pertinent to rewrite this paragraph to address the following issues:
- Punctuation: missing dot after 'communities', missing apostrophe after "organization".
- Style: starting two consecutive sentences using "It is".
- Capitalization: capitalizing the word "Decentralized".
- Precision: smart contracts are not "encoded" in a single computer, but in multiple computers that are coordinated in a decentralized network. 

Please consider requesting peer reviews to your friends and colleagues before submission.

Reply:
This initial section should also introduce the acronym "Dapp".

Reply:
Thanks @cold-briu 
I've revamped the entire piece, and it is now Grammarly proof, correcting all punctuation and grammar abberations

-  I've introduced the Dapp acronym in the introduction. Although, as you mentioned, I've made changes throughout the piece to suit the needs of a reader who already understands what a DAO is and focuses more on the sway language
- Corrected the concurrent use of it is 
- I've Resolved Capitalization problems
- I've added more context to clarify how the sharding works in a few basic terms